controllers/authenticaion.js

```diff
@@ -0,0 +1,58 @@
+require('dotenv').config();
+const customer = require('../models').customer;
+
+
+// JWT implementation
+const key = `${process.env.SECRET_JWT}`;
+let jwt = require('jsonwebtoken');
+
+// ecnryption
+const Cryptr = require('cryptr');
+const cryptr = new Cryptr(key);
+createUser =  async (body) => {
+    const user=await customer.create({
+        name: `${body.name}`,
+        email: `${body.email}`,
+        password: cryptr.encrypt(`${body.password}`)
+    });
+    return user;
+
+}
```

Comment:
@Wajahat-Khan No need to wrap variables into template string, I suggest
createUser =  async ({ name, email, password }) => {
    const user=await customer.create({
        name,
        email,
        password: cryptr.encrypt(password)
    });
    return user;

}